node/parachain/src/cli.rs

```diff
@@ -93,6 +119,12 @@ pub struct Cli {
 	#[structopt(flatten)]
 	pub run: RunCmd,
 
+	/// Run node as collator.
+	///
+	/// Note that this is the same as running with `--validator`.
+	#[structopt(long, conflicts_with = "validator")]
+	pub collator: bool,
```

Comment:
This should be updated on the documentation too @albertov19

Reply:
We discussed this in the pre-release meeting 10 Dec 2020, and @albertov19 confirmed he is aware of this and collator docs are not yet written. So I'll dismiss this conversation for easier review.